lib/clashom.gi

```diff
@@ -1633,7 +1633,7 @@ end);
 #F  LiftClassesEANonsolvGeneral( <H>,<N>,<NT>,<cl> )
```

Comment:
This (and similar comments) has a quite outdated argument list (but it was already wrong before your PR, so... *shrug*)

Reply:
Agreed, probably for a different PR